nodes/sros/sros.go

```diff
@@ -54,6 +54,7 @@ const (
 	configCf2                 = "config/cf2"
 	configCf1                 = "config/cf1"
 	startupCfgName            = "config.cfg"
+	envVarGrpcInsecureMode    = "SRSIM_GRPC_INSECURE_MODE"
```

Comment:
@sacckth let's call this env var `CLAB_SRSIM_GRPC_INSECURE` to denote that it is a clab env var

Reply:
since we are not comfortable to make cert-based flow the default one, we should get rid of this env var, and instead rely on 

`n.Cfg.Certificate.Issue`

https://github.com/srl-labs/containerlab/pull/2809/files#diff-ae71218e629cf2763a2702c67297cb2ade467276acff8f39973caf1a09731d94R188

currently it is forced to true in sros.go, as it was a copy from SRL, but we can remove this line and then by default the `Certificate.Issue` is false until set by the user in the topology.

This will make the insecure server by default, and when a user sets `cert.issue: true` it will create and provision the certs